packages/cli/src/commands/dbCommands/options.ts

```diff
@@ -0,0 +1,65 @@
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+
+interface Option {
+  alias?: string
+  default?: boolean | number | string
+  description: string
+  type: string
+}
+
+export const force = (): Option => ({
+  alias: 'f',
+  default: true,
+  description: 'Overwrite existing Client',
+  type: 'boolean',
+})
+
+export const name = (): Option => ({
+  default: 'migration',
+  description: 'Name of the migration',
+  type: 'string',
+})
+
+export const increment = (): Option => ({
+  description: 'Number of forward migrations to apply. Defaults to the latest',
+  type: 'number',
+})
+
+export const decrement = (): Option => ({
+  default: 1,
+  description: 'Number of backwards migrations to apply',
+  type: 'number',
+})
```

Comment:
Since these aren't shared with more than 1 command that it actually makes the code a bit harder to read.

Reply:
Fixed in https://github.com/redwoodjs/redwood/pull/1424/commits/24a38ce7b69ec47ceeb294dbfa636d9f99bad594.